analysers/analyser_merge_water_FR.py

```diff
@@ -119,7 +119,7 @@ def __init__(self, config, logger = None):
 class SubAnalyser_Merge_Water_FR(SubAnalyser_Merge_Dynamic):
     def __init__(self, config, error_file, logger, item, id, level, title, select, tags, generate_tags):
         SubAnalyser_Merge_Dynamic.__init__(self, config, error_file, logger)
-        self.def_class_missing_official(item = item, id = id, level = level, tags = ['merge'], title = title)
+        self.def_class_missing_official(item = item, id = id, level = level, tags = ['merge', 'water', 'fix:survey', 'fix:imagery'], title = title)
```

Comment:
fountain also with fix:chair

Reply:
I'm not sure if `fix:chair` makes sense here, you will either need an imagery or to go on ground to integrate these features. Isn't `fix:chair` about things you can fix by just checking Osmose information (refs, topology issues, knowledge from your brain...) ?